app/scripts/views/base.js

```diff
@@ -55,7 +55,9 @@ function (Cocktail, _, Backbone, Raven, $, p, AuthErrors,
 
     err = this._normalizeError(err);
 
-    this.logError(err);
+    if (shouldLog) {
+      this.logError(err);
```

Comment:
What's doubly fishy is [the comment mentions the exact situation this PR fixes](https://github.com/mozilla/fxa-content-server/blob/issue-2519-ephemeral-logged-twice/app/scripts/views/base.js#L485-L489).